modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Default.js

```diff
@@ -22,57 +22,61 @@ const nodeMap = {
 };
 
 export default (options) => {
-  const def = options.styles.filter(style => style.def)[0];
+  const [ def ] = options.styles.filter(style => style.def);
+
+  console.log('options.styles', options.styles);
+
+  if (!def) {
+    return;
+  }
 
   // Configuration has a default style
-  if (def) {
-    const nodeName = 'defaultNode';
-    const attrs = {
-      class: def.options.class || null
-    };
+  const nodeName = 'defaultNode';
+  const attrs = {
+    class: def.options.class || null
+  };
 
-    if (def.type === 'heading' || def.type === 'paragraph') {
-      return nodeMap[def.type].extend({
-        name: nodeName,
-        addOptions() {
-          return {
-            ...this.parent?.(),
-            HTMLAttributes: attrs,
-            levels: def.options.level ? [ def.options.level ] : null
-          };
-        }
-      });
-    }
+  if (def.type === 'heading' || def.type === 'paragraph') {
+    return nodeMap[def.type].extend({
+      name: nodeName,
```

Comment:
I'm not sure we need to set a special name to this node since it's still a paragraph.
I would personally manage it like a normal paragraph instead of a defaultNode. But I suppose there is a reason here?